CMakeLists.txt

```diff
@@ -222,4 +245,16 @@ endif()
 
 if(CUKE_ENABLE_EXAMPLES)
     add_subdirectory(examples)
+
+    #check if c++11 is needed
+    if(${Qt5Core_FOUND})
+#        find_program(QMAKE_EXECUTABLE NAMES qmake HINTS ${QTDIR} ENV QTDIR PATH_SUFFIXES bin)
+#        execute_process(COMMAND ${QMAKE_EXECUTABLE} -query QT_VERSION OUTPUT_VARIABLE QT_VERSION)
+        if(NOT ${Qt5Core_VERSION_STRING} VERSION_LESS 5.7)
+            message(STATUS "C++11 is needed from Qt version 5.7.0, building with c++11 enabled")
+            set_property(TARGET libcalcqt PROPERTY CXX_STANDARD 11)
+            set_property(TARGET calcqt PROPERTY CXX_STANDARD 11)
```

Comment:
No worries, I had to discover this the hard way myself. Unfortunately CMake's documentation is more of a reference manual than a usage manual.